tests/js/spec/views/alerts/rules/index.spec.jsx

```diff
@@ -55,8 +55,7 @@ describe('OrganizationRuleList', () => {
   it('displays list', async () => {
     const wrapper = await createWrapper();
 
-    const items = wrapper.find('Column');
-    expect(items).toHaveLength(5);
+    const items = wrapper.find('div');
```

Comment:
This is not a great selector because `div` has little specificity; you probably don't need it at all, I would replace `items` with `wrapper`.